bootloaders/mzero/Bootloader_D21/src/ASF/sam0/utils/cmsis/samd21/include/samd21e15a.h

```diff
@@ -218,10 +218,8 @@ void I2S_Handler                 ( void );
  * \brief Configuration of the Cortex-M0+ Processor and Core Peripherals
  */
 
-#if defined(LITTLE_ENDIAN) && (LITTLE_ENDIAN != 1)

```

Comment:
this header is part of bootloader, which is not included at all, these changes should all be reverted I think

Reply:
ok.  done.